src/docs/spec-model.md

```diff
@@ -192,3 +192,47 @@ If a defined extension slot has a `type_hint` other than `http://www.w3.org/2001
 | http://www.w3.org/2001/XMLSchema#datetime | Implementations MAY check that the value is a date and time value in the ISO 8601 format (`yyyy-mm-ddThh:mm:ssTZ`) |
 
 Implementations MAY decide to recognise more types and to enforce type-specific constraints. For example, an implementation could recognise the type `http://www.w3.org/2001/XMLSchema#negativeInteger` and check that the value starts with a minus sign.
+
+## Versioning
+
+Starting from version 1.1 of the specification, the `MappingSet` class has an optional slot named `sssom_version` which indicates the version of the specification that the set declares itself to be compliant with.
+
+### Versioning rules
+
+The version number of the SSSOM specification has two components: a _major_ number _X_ and a _minor_ number Y, expressed as `X.Y`.
+
+A set that is compliant with a minor version _X.Y_ is also compliant with any minor version _X.Y+n_, for any value of _n_. The opposite is not true: a set compliant with a minor version _X.Y_ may not necessarily be compliant with a minor version _X.Y-n_.
+
+A set that is compliant with a major version _X_ may not be compliant with any other major version _X+n_ or _X-n_.
+
+Therefore, an implementation that is itself compliant with version _X.Y_ SHOULD always accept a set compliant with any version _X.Y-n_. It MAY reject outright a set compliant with any version _X.Y+n_ (more recent minor version), _X-n_ (older major version), or _X+n_ (more recent major version).
+
+In other words, the SSSOM specification guarantees backwards compatibility between two versions (in that a set compliant with an older version can be used with an implementation compliant with a newer version) only insofar as only the _minor_ version has changed.
+
```

Comment:
When reading this I thought you might be doing something different than semantic versioning...but after re-reading, I am not sure.  Why not just link the user to [semantic versioning docs](https://semver.org/)?  If you need to be more specific than plain semantic versioning, or want to clarify it, it would help me understand what you want me to do if I started from a place of understanding.

Reply:
This is basically the same “spirit” as semantic versioning, but the semantic versioning document is much too complex for what we need – for example, we do not want to deal with things like patch levels (3-component version numbers, e.g. `1.0.3`) or alpha or beta versions.

In effect, what we want is “semver but with only MAJOR.MINOR version numbers”.

Sure, I would expect any developer worth their salt to be able to deduce what we want from the semver specification, but given that this document is itself a specification, I’d rather be as precise as possible.

Reply:
In a world where my dev time is limited (and salty 😄 ), learning a new method for something so familiar takes effort.  Are you sure you want me to use my effort to understand a new semantic versioning strategy especially with no hooks to get me familiar with the change you are trying to make?

e.g. "semver but with only MAJOR.MINOR version numbers" makes more sense to me (and sets the stage for me reading more about what you want to do to qualify semvar).

Reply:
I can consider adding an introductory sentence like “versioning of the SSSOM mostly follows the Semantic Versioning specification but with MAJOR.MINOR version numbers. Details follow.” if that helps people that are already familiar with semver, but that’s about as far as I am willing to go.

In particular, I do not want developers already familiar to semdev to read that and think, “oh, OK, I already know that thing so I can pass“ – No! Any reference to semver  would be _informative_ only, the real spec is self-contained in this document.